site/public/js/calendar.js

```diff
@@ -6,6 +6,100 @@
 const monthNames = ['December', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
 const monthNamesShort = ['Dec', 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
 
+/**
+ * Changes the year and updates cookies and loads the calendar
+ * @param x : int the value of the month to change to
+ * @param view_year : int year that is currently being viewed 
+ * @param view_month : int month that is currently being viewed
+ * @param view_day : int day that is currently being viewed
+ * @returns {void} : Loads the updated calendar
+ */
+function changeYear(x, view_year, view_month, view_day) {
```

Comment:
You can add `changeYear` to the exported section at the top of the file to address this. The rest of the linting issues are straightforward.